mne/decoding/time_gen.py

```diff
@@ -93,14 +94,15 @@ def __init__(self, picks=None, cv=5, clf=None, train_times=None,
         self.clf = clf
         self.predict_mode = predict_mode
         self.scorer = scorer
+        self.score_mode = score_mode
         self.picks = picks
         self.predict_method = predict_method
         self.n_jobs = n_jobs
 
     def fit(self, epochs, y=None):
         """Train a classifier on each specified time slice.
 
-        Note. This function sets the ``picks_``, ``ch_names``, ``cv_``,
+        .. note:: This function sets the ``picks_``, ``ch_names``, ``cv_``,
         ``y_train``, ``train_times_`` and ``estimators_`` attributes.
```

Comment:
Check other existing instances of .. note (e.g. by git grep) and you'll see what I mean